js/ui/form/components/field_item.js

```diff
@@ -162,27 +168,7 @@ export function renderFieldItem({
             });
         }
 
-        if(isMaterial()) {
-            const wrapperClass = '.' + FIELD_ITEM_CONTENT_WRAPPER_CLASS;
-            const toggleInvalidClass = function(e) {
-                $(e.element).parents(wrapperClass)
-                    .toggleClass(
-                        INVALID_CLASS,
-                        e.component.option('isValid') === false &&
-                        (e.component._isFocused() || e.component.option('validationMessageMode') === 'always')
-                    );
-            };
-
-            validationTargetInstance.on('optionChanged', (e) => {
-                if(e.name !== 'isValid') return;
-                toggleInvalidClass(e);
-            });
-
-            validationTargetInstance
-                .on('focusIn', toggleInvalidClass)
-                .on('focusOut', toggleInvalidClass)
-                .on('enterKey', toggleInvalidClass);
-        }
+        toggleWrapperInvalidClassInMaterial(validationTargetInstance);
```

Comment:
This row will cause double-subscribe if template is sync, won't it?

Reply:
I reverted some changes after your review and the situation should be more predictable